doc/admin.md

```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
+experience problems or bad performance.
+
+###  Remote Service Access
+
+We have internally two components to connect, the API and the engine (besides the database manager and the
+message bus). After that two internals component, we
+should connect with the the IdM GE. An administrator to verify that such
+links are available will use this information.
+
+The first step is to check that the api is up and running, for
+this purpose we can execute the following curl command, which is a
+simple GET operation:
+
+    root@fiware:~# curl http://$IP:$PORT/v1.0
+
+The variable will be the IP direction in which we have installed the murano service. This request should return the status of the server if it
+is working properly.
+
+
+In order to check the connectivity between Murano and the IdM
+GE, due to it must obtain a valid token and tenant for a user and
+organization with the following curl commands:
+
+    root@fiware:~# curl
+    -d '{"auth": {"tenantName": "<MY_ORG_NAME>",
+    "passwordCredentials":{"username": "<MY_USERNAME>", "password": "<MY_PASS>"}}}'
+    -H "Content-type: application/json" -H "Accept: application/xml"
+    http://<KEYSTONE_HOST>:<KEYSTONE_PORT>/v2.0/tokens
+
+The will be the name of my Organization/Tenant/Project predefined in the
+IdM GE (aka Keystone). The and variables will be the user name and
+password predefined in the IdM GE and finally the and variables will be
+the IP direction and port in which we can find the IdM GE (aka
+Keystone). This request should return one valid token for the user
+credentials together with more information in a xml format:
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <access xmlns="http://docs.openstack.org/identity/api/v2.0">
+      <token expires="2012-06-30T15:12:16Z" id="9624f3e042a64b4f980a83afbbb95cd2">
+        <tenant enabled="true" id="30c60771b6d144d2861b21e442f0bef9" name="FIWARE">
+          <description>FIWARE Cloud Chapter demo project</description>
+        </tenant>
+      </token>
+      <serviceCatalog>
+      …
+      </serviceCatalog>
+      <user username="fla" id="b988ec50efec4aa4a8ac5089adddbaf9" name="fla">
+        <role id="32b6e1e715f14f1dafde24b26cfca310" name="Member"/>
+      </user>
+    </access>
+
+With this information (extracting the token id), we can perform a GET
+operation to Murano service to check that the database is working. For this purpose we can execute
+the following curl commands:
+
+    curl -v -H 'X-Auth-Token: a9a861db6276414094bc1567f664084d'
+    -X GET "http://<I$P>:8082/v1/catalog/packages"
+
+The variable will be the IP direction in which we have installed the
+Murano API functionality. This request should return the packages already
+uploaded. In case we have not upload anyone, it will just show the core library:
+
+    {
+        "packages":
+        [
+            ...
+        ]
+    }
+
+### Resource consumption
+No issues related to resources consumption have been detected.
```

Comment:
Really?

You need to write here which resources are you usually using in Murano, if I could remember just a top. And make differentiation between normal and high demand of Murano (the last one we usually gave an estimation of it).